cmd/up.go

```diff
@@ -77,6 +74,13 @@ func NewUpCmd(flags *flags.GlobalFlags) *cobra.Command {
 			if err != nil {
 				return err
 			}
+			// try to parse flags from env
```

Comment:
not sure about these changes

Reply:
I think this was a rebase issue, I've updated now but the only change should be to use a one liner err check